assets/ca-operator/ca-operator-configmap.yaml

```diff
@@ -3,5 +3,5 @@ kind: ConfigMap
 metadata:
   name: ca-operator
 data:
-  initial-ca.crt: |-
+  initial-ca.crt: |
```

Comment:
With the trailing `-` the final line feed was getting truncated. This resulted in an incorrectly formatted ca when `cat`ed by the ca-opertor pod.